logicrunner/executionbroker.go

```diff
@@ -412,7 +412,6 @@ func (q *ExecutionBroker) OnPulse(ctx context.Context) []payload.Payload {
 		// clean everything, just in case
 		q.finished = nil
 		q.deduplicationTable = make(map[insolar.Reference]bool)
-		q.pending = insolar.InPending
 		q.PendingConfirmed = true
```

Comment:
@bigbes need your review for this change. I'm pretty sure now there is no need to to it

Reply:
disregard that